content/docs/user-guide/experiment-management/running-experiments.md

```diff
@@ -72,6 +72,13 @@ To display and compare multiple experiments along with their
 Use `dvc exp apply` to restore the results of any other experiment instead. See
 [Bring experiment results to your workspace][apply] for more info.
 
+<admon type="warn">
+
+Only files tracked by either Git or DVC are saved to the experiment. Untracked
```

Comment:
There's already a similar note in the **Running the pipeline(s)** section:

> ⚠️ Note that any changed dependencies are committed to the DVC cache when preparing the experiment...

Maybe replace it or combine them instead?

Reply:
I think they are pretty different (one is warning about exp prep being slow, the other is about losing untracked files), but I can try to combine into one admon if it looks cleaner.

Reply:
Up to you. I just think that this note is more useful at the moment of running and/or queueing that under **Experiment results**.

Reply:
I see your point, but I'm also not liking them combined because it's too much text. It's not great, but I'm going to merge as is rather than spend more time on it.